tests/nlu/extractors/test_regex_entity_extractor.py

```diff
@@ -86,12 +86,17 @@ def test_process(
     training_data.lookup_tables = lookup
     training_data.training_examples = [
         Message(
-            data={TEXT: "Hi Max!", "entities": [{"entity": "person", "value": "Max"}]}
+            data={
+                TEXT: "Hi Max!",
+                INTENT: "intent",
```

Comment:
the requirement to have an intent on an example to do entity extraction doesn't sound right, esp. for components that don't rely on that data - are we sure this is the right way to implement this?

Reply:
it depends. I don't think new data format allows to define examples with entities without intent, no?

Reply:
I was first with Tom, but then after going through the code it makes sense. intents are labels and entities are labels during the training, too. Either we have a labeled training example or not. But a mix wouldn't make sense.